src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2837,19 +2837,13 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="RestoreSelection" xml:space="preserve">
     <value>Restore</value>
   </data>
-  <data name="ShortcutDialog.PrimaryButtonText" xml:space="preserve">
-    <value>Yes</value>
-  </data>
-  <data name="ShortcutDialog.SecondaryButtonText" xml:space="preserve">
-    <value>No</value>
-  </data>
   <data name="ShortcutDialog.Title" xml:space="preserve">
     <value>Problem with Shortcut</value>
```

Comment:
"Shortcut cannot be opened"

Reply:
![image](https://user-images.githubusercontent.com/79826944/194207609-50b563dd-3754-4124-9940-6775084d879c.png)